myriad-scheduler/src/main/java/com/ebay/myriad/scheduler/fgs/OfferUtils.java

```diff
@@ -0,0 +1,35 @@
+package com.ebay.myriad.scheduler.fgs;
+
+import java.util.Collection;
+import org.apache.hadoop.yarn.api.records.Resource;
+import org.apache.mesos.Protos;
+import org.apache.mesos.Protos.Offer;
+
+/**
+ * Utility class that provides useful methods that deal with Mesos offers.
+ */
+public class OfferUtils {
+
+  /**
+   * Transforms a collection of mesos offers into {@link Resource}.
+   *
+   * @param offers collection of mesos offers
+   * @return a single resource object equivalent to the cumulative sum of mesos offers
+   */
+  public static Resource getYarnResourcesFromMesosOffers(Collection<Offer> offers) {
+    double cpus = 0.0;
+    double mem = 0.0;
+
```

Comment:
@smarella @DarinJ Sorry for my slow response.. Glad this got merged in anyway.
Mesos translates using "1" cpu resource of a total of "8" cpu resources into relative cpushares, so "1 cpu" does not mean you will get pinned to a specific core, and you are even likely get more than your requested cpu percentage, if there are idle cpus. Only during a cpu-intensive period will your task get throttled to its requested share. As such, "0.1 cpus" could be a reasonable cpu request, especially for best effort jobs.